packages/dialect-react-ui/components/NotificationsButton/index.tsx

```diff
@@ -65,8 +66,8 @@ function WrappedNotificationsButton(props: PropTypes): JSX.Element {
         icon={<icons.bell className={clsx('dt-w-6 dt-h-6 dt-rounded-full')} />}
         onClick={ui?.open ? close : open}
       />
-      <NotificationModal
-        ref={(el) => {
+      <NotificationsModal
+        ref={(el: any) => {
```

Comment:
Is it inferring types incorrectly? Why `any`?

Reply:
sorry it's typo, removed